### PR TITLE
all sort of fixes, and merged master into this branch

### PR DESCRIPTION
**Pull request by SpaceEEC**
**Please describe the changes this PR makes and why it should be merged:**

small stuff here and there, also merged master

**Semantic versioning classification:**  
- [ ] This PR changes the library's interface (methods or parameters added)
  - [ ] This PR includes breaking changes (methods removed or renamed, parameters moved or removed)
- [ ] This PR **only** includes non-code changes, like changes to documentation, README, etc.

### CODE REVIEW

File: src/structures/Invite.js [hunk 1]
```diff
@@ -154,7 +145,7 @@ class Invite {
    * @returns {Promise<Invite>}
    */
   delete(reason) {
-    return this.client.api.invites(this.code).delete({ reason }).then(() => this);
```
Comment (devsnek) [inline src/structures/Invite.js]: invites and invite go to the same place
https://discordapp.com/api/v7/invite/discord-api
https://discordapp.com/api/v7/invites/discord-api

but you didn't add `invites` to the function list

File: src/structures/ClientUser.js [hunk 2]
```diff
@@ -282,33 +282,32 @@ class ClientUser extends User {
...
+    return this.client.api.users('@me').mentions.get({ query: options })
     .then(data => data.map(m => new Message(this.client.channels.get(m.channel_id), m, this.client)));
   }
 
   /**
    * Creates a guild.
    * <warn>This is only available when using a user account.</warn>
    * @param {string} name The name of the guild
-   * @param {string} region The region for the server
+   * @param {string} [region] The region for the server
```
Comment (devsnek) [inline src/structures/ClientUser.js]: region isn't optional

**Review by devsnek — CHANGES_REQUESTED** [review]

File: src/structures/Invite.js (hunk 1 above)
Comment (SpaceEEC) [reply src/structures/Invite.js]: whups

File: src/structures/ClientUser.js (hunk 2 above)
Comment (SpaceEEC) [reply src/structures/ClientUser.js]: I could create guilds with an user account and a bot account with just a name, seems pretty optional to me.

Comment (devsnek) [reply src/structures/ClientUser.js]: Then following standard put icon and region in an object

**Review by devsnek — APPROVED** [review]

**Pull request merged**